ui/components/app/transaction-list-item-details/transaction-list-item-details.component.js

```diff
@@ -4,6 +4,7 @@ import copyToClipboard from 'copy-to-clipboard';
 import { getBlockExplorerLink } from '@metamask/etherscan-link';
```

Comment:
I think it might be best to move the transaction list related changes to a separate PR

Reply:
I will be happy to do that, how do you want to deal with that considering the timing constraints ?

Reply:
We decided not to do this